src/xdocs/checks/coding/returncount.xml

```diff
@@ -48,45 +48,41 @@
               <td>Specify method names to ignore.</td>
               <td><a href="../../property_types.html#Pattern">Pattern</a></td>
               <td><code>&quot;^equals$&quot;</code></td>
-              <td>3.4</td>
```

Comment:
What happened here?

Reply:
3.4 -> 3.2
I doubt we will discover what is right value.
Check is from 3.2 https://github.com/checkstyle/checkstyle/blob/7e668593d0f88e6a549d71e0db931d2b1b16687c/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java#L98-L102

Property
https://github.com/checkstyle/checkstyle/blob/7e668593d0f88e6a549d71e0db931d2b1b16687c/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java#L156-L158

So 3.4 is better value.

Reply:
Let's address it separately https://github.com/checkstyle/checkstyle/issues/13934
As this version is so old, I propose to allow regression in and not block migration until this issue.
And I think I should pause migration of others and fix all issues we already found.